tsdb/engine/tsm1/engine.go

```diff
@@ -184,7 +184,10 @@ func (e *Engine) Close() error {
 }
 
 // SetLogOutput is a no-op.
-func (e *Engine) SetLogOutput(w io.Writer) {}
+func (e *Engine) SetLogOutput(w io.Writer) {
```

Comment:
Again, either need a note on usage (that `SetLogOutput` should not be called once the shard is open) or we're going to need a lot more locking around access to `e.logger`.